src/js/components/FormField/FormField.js

```diff
@@ -380,6 +394,14 @@ const FormField = forwardRef(
           if (contextOnBlur) contextOnBlur(event);
           if (onBlur) onBlur(event);
         }}
+        onChange={event => {
+          const debouncedFn =  debounce(() => {
+            if (contextOnChange) contextOnChange(event);
```

Comment:
I think we need to `event.persist()` before using it outside of the immediate callback function.